src/components/30-organisms/testimonials/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axa-ch/testimonials",
-  "version": "0.0.1-alpha.0",
+  "version": "1.0.0",
```

Comment:
Because lerna manage the versions? I dont want to forget to set version to 1.0 at release time - so i manually set it. But youre right... not that nice here.